src/resolve/resolve-tool.c

```diff
@@ -840,7 +840,7 @@ static int resolve_openpgp(sd_bus *bus, const char *address) {
         }
         domain++;
 
-        r = string_hashsum_sha224(address, domain - 1 - address, &hashed);
+        r = string_hashsum_sha256(address, domain - 1 - address, &hashed);
```

Comment:
`hashed` needs to be truncated to 28 octets later, see [RFC 7929](https://tools.ietf.org/html/rfc7929#section-3)!